src/ViewInstance.js

```diff
@@ -808,6 +821,17 @@ class ViewInstance extends React.Component {
               />
             </IfPermission>
           </IfInterface>
+
+          <IfInterface name="orders">
+            <IfPermission perm="ui-inventory.instance.createOrder">
```

Comment:
IMO it's redundant to check interface and permissions for modal. It's already checking for rendering action button that call this Modal.

Reply:
I reinsured))
Removed now